frontend/src/components/shared/WorkflowSamplesTable/WorkflowSamplesTable.tsx

```diff
@@ -84,34 +92,51 @@ function WorkflowSamplesTable({sampleIDs, columns, filterDefinitions, filterKeys
 	}
 
 	const handleTableOnChange: TableProps<any>['onChange'] = (pagination, filters, sorterResult) => {
-		if( setSortBy) {
-			if(! Array.isArray(sorterResult)) {
+		if (setSortBy) {
+			if (!Array.isArray(sorterResult)) {
 				const sorter = sorterResult
 				const key = sorter.columnKey?.toString()
 				const order = sorter.order ?? undefined
 				if (key) {
 					if (sortBy === undefined || key !== sortBy.key || order !== sortBy.order) {
-						setSortBy({key, order})
+						setSortBy({ key, order })
 					}
 				}
 			}
 		}
 	}
-	
+
 	return (
 		<>
-			{tableColumns && 
+			{tableColumns &&
 				<>
+					{
+						hasFilter &&
```

Comment:
Let's create a single stand-alone component that combines the div, FiltersWarning and Button and call it `FiltersBar` (I can't think of a better name that doesn't already conflict with the existing components :) ). 

I would also like to move the filters bar out of WorkflowSamplesTable. I think the filters bar can exist independently of the samples table. You would create a `<FiltersBar>` component in the parent component that is using the samples table (eg. LabworkStep).